cmd/frontend/graphqlbackend/org.go

```diff
@@ -231,6 +232,49 @@ func (o *OrgResolver) ViewerIsMember(ctx context.Context) (bool, error) {
 	return true, nil
 }
 
+func (o *OrgResolver) ViewerNeedsCodeHostUpdate(ctx context.Context) (bool, error) {
+	actor := actor.FromContext(ctx)
+	if !actor.IsAuthenticated() {
+		return false, nil
+	}
+	enabled, err := o.db.FeatureFlags().GetOrgFeatureFlag(ctx, o.OrgID(), "github-app-cloud")
+	if err != nil {
+		return false, err
+	} else if !enabled {
+		return false, nil
+	}
+	if _, err := o.db.OrgMembers().GetByOrgIDAndUserID(ctx, o.org.ID, actor.UID); err != nil {
+		return false, nil
+	}
+	orgServices, err := o.db.ExternalServices().List(ctx, database.ExternalServicesListOptions{Kinds: []string{extsvc.KindGitHub}, NamespaceOrgID: o.OrgID()})
+	if err != nil {
+		return false, err
+	}
+	if len(orgServices) == 0 {
+		// no need to update
+		return false, nil
+	}
+	userServices, err := o.db.ExternalServices().List(ctx, database.ExternalServicesListOptions{Kinds: []string{extsvc.KindGitHub}, NamespaceUserID: actor.UID})
+	if err != nil {
+		return false, err
+	}
+	// no need to update
+	if len(userServices) == 0 {
+		// no need to update
```

Comment:
```suggestion
	if len(userServices) == 0 {
		// no need to update
```

One of these seems redundant?